css/css-display/display-contents-accessibility.html

```diff
@@ -0,0 +1,321 @@
+        <dt>x</dt>
+        <dd>y</dd>
+    </dl>
+
+    <!-- Tables (including grids/treegrids) -->
+    <table style="display: contents;" data-expectedrole="table" data-testname="table with display:contents has table role" class="ex">
+        <thead>
+          <tr style="display: contents;" data-expectedrole="row" data-testname="tr in table with display:contents has row role" class="ex">
+            <th style="display: contents;" data-expectedrole="columnheader" data-testname="td within tr both with display:contents, in table with display:contents, has columnheader role" class="ex">x</th>
+            <th>x</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td style="display: contents;" data-expectedrole="cell" data-testname="td within tr in table, all with display:contents, has cell role" class="ex">x</td>
+            <td>x</td>
+          </tr>
+        </tbody>
+      </table>
+
+    <div role="table" style="display: contents;" data-expectedrole="table" data-testname="div with role table with display:contents has table role" class="ex">
+        <div role="rowgroup" style="display: contents;" data-expectedrole="rowgroup" data-testname="div with role rowgroup in div with role table, both with display:contents, has rowgroup role" class="ex">
+            <div role="row">
+                <div role="columnheader" style="display: contents;" data-expectedrole="columnheader" data-testname="div with role columnheader with display:contents, in div with role rowgroup within div with table role both with display:contents, has columnheader role">x</div>
+                <div role="columnheader">x</div>
+            </div>
+        </div>
+        <div role="rowgroup">
+            <div role="row">
+                <div role="cell"></div>
+                <div role="cell" style="display: contents;" data-expectedrole="cell" data-testname="div with cell role and display:contents, within div with role row contained in a div with role table both with display:contents, has cell role" class="ex"></div>
+            </div>
+        </div>
+    </div>
+
+    <table role="grid" style="display: contents;" data-expectedrole="grid" data-testname="table with role grid and display:contents has grid role">
+        <thead>
+          <tr style="display: contents;" data-expectedrole="row" data-testname="tr within table with role grid, both with display:contents, has row role">
+            <th style="display: contents;" data-expectedrole="columnheader" data-testname="th within table with role grid, both with display:contents, has columnheader role">x</th>
+            <th>x</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td style="display: contents;" data-expectedrole="cell" data-testname="td within table with role grid, both with display:contents, has cell role" class="ex">x</td>
```

Comment:
I don't recall offhand whether the ARIA spec allows static `cell` (instead of `gridcell`) inside of a dynamic `grid` like this one, but given that there are multiple failures (Chromium and Gecko both return `gridcell`) please double-check that before merging.

Reply:
@cookiecrook You're correct; for the [grid](https://w3c.github.io/aria/#grid) role:

> The cell elements of a grid have role [gridcell](https://w3c.github.io/aria/#gridcell).

(If you think we should modify the language to state "must have a role `gridcell`", I can submit an issue).

Updated this test to use `gridcell`: https://github.com/web-platform-tests/wpt/pull/43740/commits/6f948421c105dcfef67d29aad96e85943c986351.